basic_portfolio/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Portfolio Page</title>
+<style>
+h1{
+	background-color: lavender;
+	height: 20%;
+	text-align: right;
+}
+#headertext li {
+	display: inline;
+	list-style: none;
+}
+div {
+	height: 60%;
+}
+footer {
+	background-color: lavender;
+}
+#footertext li {
+	display: inline;
+	list-style: none;
+	width: 10%;
+	text-align: center;
+}
+</style>
+</head>
+<body>
+	<h1>
+		<img src="http://news.toyark.com/wp-content/uploads/sites/4/2015/09/Koto-Force-Awakens-Stormtroopers-001-150x150.jpg"/>
```

Comment:
due to the normal flow of the document, this would automatically be in the top left for now.  for the logo, though, I might want to put a class of some sort so you can style it if needed.